docs/using_packet.md

```diff
@@ -29,7 +29,7 @@ version = "0.1.0"
 authors = ["My Name <my.email@mydomain.com>"]
 build = "build.rs"
 [build-dependencies]
-syntex = "0.31.*"
+syntex = "0.42.*" # syntex version must match the version being used in pnet_macros/Cargo.toml
```

Comment:
Seems like the kind of thing that would be nice to have as part of a release script at some point in the future - we could automatically update the document with the right version.